docs/docfx.json

```diff
@@ -90,13 +152,17 @@
           "**/includes/**"
         ],
         "src": "docs-ref-conceptual",
-        "version": "office-js",
         "dest": "office"
       }
     ],
-    "versions": {
-      "office-js": {
-        "dest": "office-js"
+    "groups": {
```

Comment:
`groups` names should match `moniker_ranges` in `.openpublishing.publish.config.json`